js/apps/admin-ui/src/realm-settings/RealmSettingsTabs.tsx

```diff
@@ -71,6 +72,9 @@ const RealmSettingsHeader = ({
   const [partialImportOpen, setPartialImportOpen] = useState(false);
   const [partialExportOpen, setPartialExportOpen] = useState(false);
 
+  const { hasAccess } = useAccess();
+  const isManageRealm = hasAccess("manage-realm");
```

Comment:
Perhaps this would be more appropriately named `canManageRealm`?

```suggestion
  const canManageRealm = hasAccess("manage-realm");
```

Reply:
Done! :wink: